frontend/src/pages/Workflows/WorkflowContainer.js

```diff
@@ -126,6 +130,12 @@ class WorkflowContainer extends Component {
           />
           <SubprojectHistoryDrawer projectId={this.projectId} subprojectId={this.subprojectId} />
           <WorkflowBatchEditContainer projectId={this.projectId} subProjectId={this.subprojectId} />
+          <InformationDialog
+            dialogShown={this.props.rejectReasonDialogContent ? true : false}
+            title={strings.workflow.workflow_reject_reason}
+            content={this.props.rejectReasonDialogContent ? <Typography>{this.props.rejectReasonDialogContent}</Typography> : ''}
```

Comment:
I dont thing an extra variable is necessary here. The rejectReason should be in workflowItems[] if its rejected. This is fetched on FETCH_ALL_SUBPROJECT_DETAILS_SUCCESS